assembly/resources/README.txt

```diff
@@ -0,0 +1,48 @@
+This is a distribution of
+
+       Camunda 7 Data Migrator v${project.version}
+
+visit
+       http://docs.camunda.org/
+
+==================
+
+Usage
+
+To run the Camunda 7 Data Migrator:
+
+1. Adjust `./configuration/application.yml`  
+   - The `source` config should point to your Camunda 7 database  
+   - The `target` config should point to your Camunda 8 database
+
+2. Execute the `start` script:  
+   - `start.bat` for Windows  
+   - `start.sh` for Linux/macOS
+
+Options
+
+- `--runtime` – Migrate runtime data only  
+- `--history` – Migrate history data only 
+
+==================
```

Comment:
/suggest Would be cool to have a more general get started guide in the `README.md` in the root of this project.

Reply:
Yana already updated the README addressing some of this points. I will create a new PR to update based on the rest. I am creating a new PR as the README changes are already in main.